package.json

```diff
@@ -167,6 +167,7 @@
     "cheerio": "^1.0.0-rc.2",
     "currency.js": "^1.2.1",
     "farce": "^0.2.6",
+    "flickity": "^2.2.0",
```

Comment:
In the follow-up PR removing `react-flickity-component` this can be moved to `devDependencies and added to `peerDependencies`. We can then use Force's version.